boards/feather-m0-wifi/Kconfig

```diff
@@ -10,14 +10,7 @@ config BOARD
 config BOARD_FEATHER_M0_WIFI
     bool
     default y
-    select CPU_MODEL_SAMD21G18A
-    select HAS_PERIPH_ADC
-    select HAS_PERIPH_I2C
-    select HAS_PERIPH_PWM
-    select HAS_PERIPH_RTC
-    select HAS_PERIPH_RTT
-    select HAS_PERIPH_SPI
-    select HAS_PERIPH_TIMER
-    select HAS_PERIPH_UART
-    select HAS_PERIPH_USBDEV
-    select HAS_HIGHLEVEL_STDIO
+    select BOARD_FEATHER_M0
+    select MODULE_ATWINC15X0 if TEST_KCONFIG && MODULE_NETDEV_DEFAULT
```

Comment:
We are taking a different approach to this. We use symbols to allow boards express which hardware is onboard, and then use them from the drivers. This limits the responsibility of boards to just describe themselves in most cases.

https://github.com/RIOT-OS/RIOT/blob/85313ccc02cdb813d65f208c56d06992ad484130/boards/avr-rss2/Kconfig#L25

https://github.com/RIOT-OS/RIOT/blob/85313ccc02cdb813d65f208c56d06992ad484130/drivers/bmx280/Kconfig#L9-L25

Reply:
OK, I see. I will change.

Reply:
Fixed with 8dcea85e12d805d184ef1f5e052d017931a8d9dd and ba05772434c84b5a09f50e05659b538bc7fb6d6f.